src/sentry/integrations/slack/actions/notification.py

```diff
@@ -90,7 +90,9 @@ def render_label(self) -> str:
             workspace=self.get_integration_name(),
             channel=self.get_option("channel"),
             channel_id=self.get_option("channel_id"),
-            tags="[{}]".format(", ".join(tags)),
+            tags=" and show tags " + "[{}]".format(", ".join(tags)) + " in notification"
+            if tags != [""]
+            else "",
```

Comment:
Move ` and show {tags} in notification` text from label to `render_label` function so that it can not be included if tags is equal to `[""]` (ie. no tags are selected)